src/sentry/api/endpoints/organization_events.py

```diff
@@ -32,16 +31,12 @@ def get(self, request, organization):
             params = self.get_filter_params(request, organization)
             snuba_args = self.get_snuba_query_args(request, organization, params)
             fields = snuba_args.get('selected_columns')
+            aggregations = snuba_args.get('aggregations')
             groupby = snuba_args.get('groupby', [])
 
-            if not fields and not groupby:
+            if not fields and not aggregations and not groupby:
                 return Response({'detail': 'No fields or groupings provided'}, status=400)
 
-            if any(field for field in groupby if field not in ALLOWED_GROUPINGS):
-                message = ('Invalid groupby value requested. Allowed values are ' +
-                           ', '.join(ALLOWED_GROUPINGS))
-                return Response({'detail': message}, status=400)
```

Comment:
This was removed because the endpoint no longer allows the end user to define groupby. Instead groupings are implicit based on the fields requested.